src/ontology/cl-edit.owl

```diff
@@ -30775,6 +30776,16 @@ AnnotationAssertion(rdfs:label obo:CL_4033014 "peg cell")
 SubClassOf(obo:CL_4033014 obo:CL_0000066)
 SubClassOf(obo:CL_4033014 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0004804))
 
+# Class: obo:CL_4033015 (retinal astrocyte)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:25236977") Annotation(oboInOwl:hasDbXref "PMID:33796062") obo:IAO_0000115 obo:CL_4033015 "A star-shaped glial cell that is part of some retina. This cell links neurons to blood vessels and provides structural and physiological support to optic nerve head axons. In response to elevated intraocular pressure, a retinal astrocyte modulates extracellular matrix remodelation.")
```

Comment:
Consider:

"... This cell links neurons to blood vessels and may provide structural and physiological support to optic nerve head axons. In response to elevated intraocular pressure, a retinal astrocyte may modulate extracellular matrix remodeling."

The final sentence could also be placed in a comment if you'd like to have a leaner text definition.